styles/style.scss

```diff
@@ -181,7 +186,7 @@ article {
   padding-bottom: 60px;
 
   // Make first paragraph on content pages use leadText styles
-  p:first-of-type {
+  div > p:first-of-type {
```

Comment:
I see now.  I think it’s okay but it would be better to make it more specific to the page markup to ensure it only gets applied to the first paragraph at the page. I can check this out next week.

Reply:
Would be awesome! Thanks